core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -84,6 +87,7 @@
   private boolean fullStreamDecompression;
   private DecompressorRegistry decompressorRegistry = DecompressorRegistry.getDefaultInstance();
   private CompressorRegistry compressorRegistry = CompressorRegistry.getDefaultInstance();
+  private final Set<Cork> corks = Sets.newConcurrentHashSet();
```

Comment:
We cannot take a dependency on Guava collections in the client side code because it bloats Android jars too much.

Reply:
Yes, please avoid pulling in Guava collections - it's overhead is significant on Android (unfortunately the Android increase in method counts doesn't show up much from our android/dex_diff CI check right now, because of a transitive dependency already bringing in Guava collections, but that dependency is being fixed in https://github.com/grpc/grpc-java/pull/4494)

Reply:
fixed